bitnami/argo-workflows/templates/server/clusterrolebinding.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if and .Values.server.enabled .Values.server.serviceAccount.create -}}
```

Comment:
I'd add an rbac.create value just like the other charts

Reply:
Also, I dont' see why we need serviceAccount.create to be true. We could be using an existing serviceAccount, that's what the `argo-workflows.server.serviceAccountName` is here for